src/main/java/org/prebid/server/proto/openrtb/ext/response/ExtBidResponsePrebid.java

```diff
@@ -19,4 +20,6 @@ public class ExtBidResponsePrebid {
      * Defines the contract for bidresponse.ext.prebid.modules
      */
     ExtModules modules;
+
+    JsonNode passthrough;
```

Comment:
You updated code in other classes, check this one too
 @Value(staticConstructor = "of")